types/apple-music-api/index.d.ts

```diff
@@ -130,7 +135,7 @@ declare namespace AppleMusicApi {
     // https://developer.apple.com/documentation/applemusicapi/artist/relationships
     interface ArtistRelationships {
         albums: Relationship<Album>;
-        genres: Relationship<Genre>;
+        genres?: Relationship<Genre>;
```

Comment:
I changed this to be optional because it seems the API didn't return this for 5/5 of the artists that I tried.